osm-seed/templates/web/web-service.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     environment: {{ .Values.environment }}
     release: {{ .Release.Name }}
   annotations:
+    {{- if eq .Values.serviceType "LoadBalancer" }}
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "300"
```

Comment:
Do we still need this?

Reply:
Yea, we used to have this setting - changed with an osm-seed update - I think its a good idea to keep it around.